### PR TITLE
[networkx] remove experimental tag of nx module

### PR DESCRIPTION
**Pull request by acezen**
## What do these changes do?
- rename macro definition `EXPERIMENTAL_ON` to `NETWORKX` in engine side and make it default ON.
- remove experimental tag of nx module 
- add install folly dependency in install_dependencies.sh

## Related issue number
#62

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ VERSION     	 	  			?= 0.1.0
 INSTALL_PREFIX      		?= /usr/local
 
 # GAE build options
-EXPERIMENTAL_ON     		?= OFF
+NETWORKX     		?= OFF
```
Comment (sighingnow) [inline Makefile:8]: Indentation.

**Review by sighingnow — APPROVED** [review]
LGTM.

File: analytical_engine/CMakeLists.txt [hunk 2]
```diff
@@ -17,10 +17,11 @@ if (POLICY CMP0048)
...
 project(analytical_engine LANGUAGES C CXX VERSION ${GRAPHSCOPE_ANALYTICAL_VERSION})
 
-option(EXPERIMENTAL_ON "experimental on?" OFF)
+option(NETWORKX "networkx on?" ON)
 option(BUILD_TESTS "Build unit test" ON)
 
-if (EXPERIMENTAL_ON)
+if (NETWORKX)
+    add_definitions(-DNETWORKX)
     add_definitions(-DEXPERIMENTAL_ON)
```
Comment (siyuan0322) [inline analytical_engine/CMakeLists.txt:25]: This `EXPERIMENTAL_ON` can be deleted?

**Review by siyuan0322 — APPROVED** [review]

File: analytical_engine/CMakeLists.txt (hunk 2 above)
Comment (sighingnow) [reply analytical_engine/CMakeLists.txt:25]: Vineyard v0.2.1 still requires `EXPERIMENTAL_ON`.

Comment (acezen) [reply analytical_engine/CMakeLists.txt:25]: I will add a TODO comment here

**Pull request merged**